contribs/gmf/examples/xsdattributes.html

```diff
@@ -34,6 +34,10 @@
             ng-options="layer.name for layer in ctrl.layers">
             <option value="">-- Choose a layer --</option>
           </select>
+          <div>
+            Geometry type:
+            <span>{{ ctrl.attributes ? ctrl.getGeomType() : 'N/A' }}</span>
```

Comment:
`ctrl.getGeomType()` already returns `'N/A`, `{{ ctrl.getGeomType() }}` should be good enough.